BoxContentSDKSampleApp/Podfile.lock

```diff
@@ -1,21 +1,21 @@
 PODS:
-  - box-ios-sdk (1.0.11):
-    - box-ios-sdk/logger (= 1.0.11)
-    - box-ios-sdk/no-arc (= 1.0.11)
-  - box-ios-sdk/logger (1.0.11)
-  - box-ios-sdk/no-arc (1.0.11):
+  - box-ios-sdk (1.0.14):
+    - box-ios-sdk/logger (= 1.0.14)
+    - box-ios-sdk/no-arc (= 1.0.14)
+  - box-ios-sdk/logger (1.0.14)
+  - box-ios-sdk/no-arc (1.0.14):
     - box-ios-sdk/logger
 
 DEPENDENCIES:
   - box-ios-sdk (from `../`)
 
 EXTERNAL SOURCES:
   box-ios-sdk:
-    :path: ../
+    :path: "../"
 
 SPEC CHECKSUMS:
-  box-ios-sdk: a34a7c130e83e3d2380d2f6c1f16e589ba7788ba
+  box-ios-sdk: 8f8663807e26fd5e6bb89a07c0402d2bef605d30
 
-PODFILE CHECKSUM: 321973ea596897514e372fa19af769c896c0fe55
+PODFILE CHECKSUM: 665c09eb2d2e131c961508f8a52594e06e23dab9
 
-COCOAPODS: 1.0.0
+COCOAPODS: 1.2.0
```

Comment:
1.2.0 is the current latest stable version of CocoaPods. It'll be best for devs who are keeping up with the latest.